WordPress/Classes/ViewRelated/AccountSettingsViewController.swift

```diff
@@ -27,66 +29,109 @@ private struct AccountSettingsController: SettingsController {
 
     // MARK: - Initialization
 
-    let service: AccountSettingsService
+    let service: AccountSettingsService?
 
-    init(service: AccountSettingsService) {
+    init(service: AccountSettingsService?) {
         self.service = service
     }
-    
-    // MARK: - Model mapping
 
-    func mapViewModel(settings: AccountSettings?, presenter: ImmuTablePresenter) -> ImmuTable {
-        let username = TextRow(
-            title: NSLocalizedString("Username", comment: "Account Settings Username label"),
-            value: settings?.username ?? "")
+    // MARK: - ImmuTableViewController
 
-        let email = TextRow(
-            title: NSLocalizedString("Email", comment: "Account Settings Email label"),
-            value: settings?.email ?? "")
+    func tableViewModelWithPresenter(presenter: ImmuTablePresenter) -> Observable<ImmuTable> {
+        if let service = self.service {
+            return service.settings.map({ settings in
```

Comment:
Same as above, since `settings` is a single instance, are we sure we wanna use `map` instead of `if let` / `guard`?.